visualization/app/codeCharta/ui/codeMap/codeMap.component.scss

```diff
@@ -9,4 +9,25 @@ code-map-component {
 			right: 350px;
 		}
 	}
+
+	.action-buttons {
```

Comment:
When you are already working on that button class:
- Do you want to simplify the CSS? Can we maybe get almost rid of it by using the `small-action-button` class?
- Shouldn't we move that styling to the `unfocusButton.component.scss`

Reply:
I agree with the Unfocus description.

Right now there is only one action-button, so the class `action-buttons` is not really meaning-ful. Instead let's move the css to the unfocus-button-component as suggested.

Reply:
Done

Reply:
what we can do as well is, if you right-click the focused building, that you can instead unfocus it.